src/pudl_archiver/archivers/phmsagas.py

```diff
@@ -77,7 +77,7 @@ async def get_zip_resource(
         if form not in PHMSA_FORMS:
             logger.warning(f"New form type found: {form}.")
 
-        download_path = self.download_directory / f"{self.name}-{filename}.zip"
+        download_path = self.download_directory / f"{self.name}_{filename}.zip"
```

Comment:
We'll need to fix this in the PUDL excel files once this gets merged and we move to a new archive, but it's a weird inconsistency that's been bothering me and I want to just fix it.